test/end-to-end/pages/createImage.js

```diff
@@ -20,7 +20,7 @@ module.exports = class CreateImagePage extends MainPage {
     this.labelCreateImage = `${this.dialogRootElement} #myModalLabel`;
 
     // Blueprint Name label
-    this.labelBlueprintName = `${this.dialogRootElement} p[class="form-control-static"]`;
+    this.labelBlueprintName = `${this.dialogRootElement} #cmpsr-form-blueprint`;
 
     // Image Type select
     this.selectImageType = `${this.dialogRootElement} label[for="textInput-modal-markup"] + div select`;
```

Comment:
Now that you've rebased, you can update this selector to not be so confusing.

The original selector is looking for a `<select>` inside a `<div>` that's preceded by a label with the html attribute `for="textInput-modal-markup"`. 

But this `<select>` has an `id` that we can use instead:
```
                    <select 
                      id="textInput-modal-markup" 
                      className="form-control" 
                      required 
                      value={this.state.imageType} 
                      onChange={this.handleChange}
                    >
```
So now we can simplify the selector to:
```
this.selectImageType = `${this.dialogRootElement} #textInput-modal-markup`;
```